pandas/core/indexes/interval.py

```diff
@@ -1039,43 +1039,37 @@ def interval_range(start=None, end=None, freq=None, periods=None,
         Left bound for generating data
     end : string or datetime-like, default None
         Right bound for generating data
-    freq : interger, string or DateOffset, default 1
-    periods : interger, default None
+    freq : integer, string or DateOffset, default 1
```

Comment:
somewhere mention (if not already) that freq must be consistent with the type of the start/end

Reply:
also I *think* there are tests but for this, but pls confirm.

Reply:
Confirmed that there are tests for mixed units in `TestIntervalRange.test_errors`.  Adding one additional test to be safe.